src/docs/background/21-authentication-api/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Authentication API
+slug: authentication-api
+updated: 2019-11-27
+category: background
+ingress:
+  Description of the Flex Authentication API and how the
+  Marketplace API and Integration API applications use it
+published: true
+---
+
+## Authentication API
+
+Both the [Marketplace API and the Integration
+API](/background/marketplace-api-integration-api/) require valid *access tokens*
+to be passed in every API request. Applications obtain those access tokens from
+the Authentication API.
+
+As a general rule, applications that access the Marketplace API do so by
+authenticating an end user of the marketplace (via the user' username and
+password), while Integration API applications authenticate using their own
+credentials.
+
+In order to access Flex APIs, you need to create an *Application* in [Flex
+Console](https://flex-console.sharetribe.com/applications). Each Application has
+a *client ID*. In addition, applications that access the Integration API also
+have a corresponding *client secret*.
+
+> **NOTE** The easiest way to interact with both the Marketplace API and the
+> Integration API is through the corresponding [SDKs](/references/js-sdk/). The
+> SDKs handle most of the complexity regarding authentication, access and
+> refresh tokens. Below we discuss some of the underlying mechanisms and
+> principles in the Authentication API.
+
+The Authentication API is based on the [OAuth 2.0](https://oauth.net/2/)
+framework. While the API does not implement many of the mechanisms allowed in
+the framework, it does largely follow the specification with the endpoints and
+mechanisms it supports.
+
```

Comment:
We could sell this a bit more boldly and e.g. drop the "while the API does not implement many...". I think that's the common case anyway to implement a subset of endpoints and grant types.